src/Templates/Fullscreen/Fullscreen.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Dialog } from '@material-ui/core'
+import {
+  Button, IconButton, Tooltip, Typo, Trans,
+} from '../../Atoms'
+import { X } from 'styled-icons/feather/X'
+import { Question } from '@styled-icons/evil/Question'
+import FlexBox from '../FlexBox'
+import { useTranslation } from '../../Atoms/Trans'
+
+const useStyles = makeStyles(theme => {
+  return {
+    root: {
+      backgroundColor: 'white',
+      position:        'absolute',
+      top:             0,
+      left:            0,
+      height:          '100%',
+      width:           '100%',
+      '& .close':      {
+        position: 'absolute',
+        top:      10,
+        right:    80,
+      },
+      '& .buttons': {
+        position: 'absolute',
+        bottom:   47,
+        right:    80,
+        '& > *':  {
+          marginLeft: 7,
+        },
+      },
+      '& .title': {
+        textTransform: 'uppercase',
+        '& span':      {
+          marginLeft: 4,
+        },
+      },
+      '& .header, .body': {
+        marginLeft: 67,
+        marginTop:  22,
+      },
+      '& .subTitle > *': {
+        marginLeft: 3,
+      },
+      '&.hidden': {
+        visibility: 'hidden',
+      },
+    },
+  }
+})
+
+const Fullscreen = ({
+  icon, headerTitle, title, tooltip, children, setOpen, onValidate, validateText, ...props
```

Comment:
I think you should replace setOpen with isOpen/open here, so the full visibility is given to the caller instead